### PR TITLE
Replacing beginner video with Limon's

### PR DESCRIPTION
**Pull request by kensherman**
Replaces bardwells 4.3 betaflight series of videos with Limons

### CODE REVIEW

File: src/pages/index.ejs [hunk 1]
```diff
@@ -203,7 +203,9 @@ description: "Betaflight is flight controller software (firmware) used to fly mu
 							<div class="container">
 								<div class="video-content">
 			
-									<iframe width="560" height="315"  src="https://www.youtube.com/embed/videoseries?list=PLwoDb7WF6c8nT4jjsE4VENEmwu9x8zDiE" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+									<div class="video-content">
```
Comment (Asizon) [inline src/pages/index.ejs:206]: This div and class are already defined and opened

**Review by Asizon — CHANGES_REQUESTED** [review]

File: src/pages/index.ejs (hunk 1 above)
Comment (kensherman) [reply src/pages/index.ejs:206]: The previous rev with the 4.2 videos used this presentation as it was a single video. When I put bardwells videos in they are a playlist which can only be embedded though an iframe. This help video is now back to a single video which I felt suited the oringal styling. My sense is either one works usability wise and I can go either way.

**Review by sugaarK — APPROVED** [review]

File: src/pages/index.ejs (hunk 1 above)
Comment (Asizon) [reply src/pages/index.ejs:206]: Good! But please remove this extra container, it already exist

**Comment by sugaarK** [comment]
Ohh lols @Asizon  my bad

**Pull request merged**